addons/auth_oauth/views/res_users_views.xml

```diff
@@ -10,7 +10,8 @@
                         <group>
                             <field name="oauth_provider_id"/>
                             <field name="oauth_uid"/>
-                            <field name="oauth_access_token"/>
+                            <field name="has_oauth_access_token" invisible="1"/>
```

Comment:
Is it still necessary `invisible="1"` nowadays in master ? Didn't Christophe (chm) implement a feature where these invisible fields because of a dependence for another field are added automatically to the view ?